ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -880,7 +879,7 @@ export const AdminInstitutionEdit = fp.flow(
                   />
                 )}
                 <TextInputWithLabel
-                  value={institution.requestAccessUrl}
+                  value={requestAccessUrl ?? ''}
```

Comment:
I think this can simply be `requestAccessUrl` - it did not seem to affect the functionality when testing locally anyway.